templating.rst

```diff
@@ -301,6 +300,23 @@ When working with template inheritance, here are some tips to keep in mind:
           {{ parent() }}
       {% endblock %}
 
+.. caution::
+
+    When using ``extends``, a child template is forbidden to define template
+    parts outside of a block. The following code throws a ``SyntaxError``:
+
+    .. code-block:: html+twig
+
+        {# app/Resources/views/blog/index.html.twig #}
+        {% extends 'base.html.twig' %}
+
+        {# the line below is not captured by a "block" tag #}
```

Comment:
```suggestion
        {# the line below is not captured by a {% block %}-tag #}
```
?

Reply:
I first went with `{% block ... %}...{% end block %}`, but I changed for the current version. I thought about what you suggest, but that's not valid syntax, a block name is required.
I think this is understandable enough as is, `"block"` in this context is just the name of a tag, and should not look like a tag expression, I guess it would be more confusing/error prone.